x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -48,10 +55,29 @@ export function isJavaAgentName(
  *
  * * Converting to lowercase
  * * Converting "rum-js" to "js-base"
+ * * Converting OpenTelemetry agent names to "our" agent names
  *
  * This helps dealing with some older agent versions
  */
 export function getNormalizedAgentName(agentName?: string) {
-  const lowercased = agentName && agentName.toLowerCase();
-  return isRumAgentName(lowercased) ? 'js-base' : lowercased;
+  let newName = agentName && agentName.toLowerCase();
+
+  if (isRumAgentName(newName) || newName === 'opentelemetry/webjs') {
+    newName = 'js-base';
```

Comment:
This whole function is garbage and I'm not proud of it and I'm going to rewrite it. :) 

I think it might not be used anywhere except in the icon selection so I'm going to try and put it in the best place.